tutorials/physics/rigid_body.rst

```diff
@@ -0,0 +1,51 @@
+.. _doc_rigid_body
+
+Rigid Body
+=========
+
+What is a rigid body?
+--------------------
+
+Rigid body is a rigid (non deformable) physical object that is directly controlled by physics engine in order to simulate phisical behaviour of objects.
+It's necessary to add one or more shapes that define the final shape of rigid body, you can set different position of these primitive shapes and this affect the COM (center of mass) of body.
```

Comment:
In order to define the shape of the body, it must have one or more :ref:`Shape <class_Shape>`objects assigned. Note that setting the position of these shapes will affect the body's center of mass.